app/src/main/java/org/kiwix/kiwixmobile/localFileTransfer/WifiDirectManager.kt

```diff
@@ -188,7 +189,7 @@ class WifiDirectManager @Inject constructor(
 
   /* From WifiP2pManager.PeerListListener callback-interface */
   override fun onPeersAvailable(peers: WifiP2pDeviceList) =
-    callbacks.updateListOfAvailablePeers(peers)
+    callbacks?.updateListOfAvailablePeers(peers)!!
```

Comment:
elvis to an empty list or whatever the return type is here? hard to tell from github

Reply:
is the return type Unit?